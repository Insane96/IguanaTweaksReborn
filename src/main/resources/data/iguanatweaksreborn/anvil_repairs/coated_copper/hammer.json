{
	"item_to_repair": {
		"id": "iguanatweaksreborn:coated_copper_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 5
		}
	]
}