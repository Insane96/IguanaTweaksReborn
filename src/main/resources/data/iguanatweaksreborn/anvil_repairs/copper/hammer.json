{
	"item_to_repair": {
		"id": "iguanatweaksreborn:stone_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 5
		}
	]
}