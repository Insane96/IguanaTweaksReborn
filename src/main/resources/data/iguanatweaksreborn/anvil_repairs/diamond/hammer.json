{
	"item_to_repair": {
		"id": "survivalreimagined:diamond_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 5
		}
	]
}