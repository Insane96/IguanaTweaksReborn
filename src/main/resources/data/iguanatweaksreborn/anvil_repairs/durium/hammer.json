{
	"item_to_repair": {
		"id": "iguanatweaksreborn:durium_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 5,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_nugget"
			},
			"amount": 18
		}
	]
}