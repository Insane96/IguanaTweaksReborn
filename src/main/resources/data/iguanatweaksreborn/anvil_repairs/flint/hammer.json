{
	"item_to_repair": {
		"id": "iguanatweaksreborn:stone_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 5
		}
	]
}