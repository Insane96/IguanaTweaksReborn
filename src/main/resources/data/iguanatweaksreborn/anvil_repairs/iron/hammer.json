{
	"item_to_repair": {
		"id": "survivalreimagined:iron_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 5
		},
		{
			"repair_material": {
				"id": "minecraft:iron_nugget"
			},
			"amount": 45
		}
	]
}