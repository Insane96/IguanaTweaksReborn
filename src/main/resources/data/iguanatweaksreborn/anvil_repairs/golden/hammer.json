{
	"item_to_repair": {
		"id": "survivalreimagined:golden_hammer"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 5
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 45
		}
	]
}